source/MRMesh/MRMeshLoadStep.cpp

```diff
@@ -31,28 +30,12 @@
 #include <opencascade/TopoDS.hxx>
 #pragma warning( pop )
 
-#if ( OCC_VERSION_MAJOR < 7 ) || ( OCC_VERSION_MAJOR == 7 && OCC_VERSION_MINOR < 5 )
-namespace MR::MeshLoad
-{
-
-Expected<std::shared_ptr<Object>, std::string> fromSceneStepFile( const std::filesystem::path& path, const MeshLoadSettings& settings /*= {}*/ )
-{
-    // TODO: support OpenCASCADE 7.3
-    return unexpected( "Unsupported: outdated OpenCASCADE version" );
-}
-
-Expected<std::shared_ptr<Object>, std::string> fromSceneStepFile( std::istream& in, const MeshLoadSettings& settings /*= {}*/ )
-{
-    return unexpected( "Unsupported: outdated OpenCASCADE version" );
-}
-
-} // namespace MR::MeshLoad
-#else
 namespace
 {
 
 using namespace MR;
 
+#if ( OCC_VERSION_MAJOR > 7 ) || ( OCC_VERSION_MAJOR == 7 && OCC_VERSION_MINOR >= 4 )
```

Comment:
mb create one define not to check whole version every time?

Reply:
Fixed.